docs/github.md

```diff
@@ -105,7 +112,7 @@ top, you can do that by doing something like this:
 ```shell
 $ # Create a new commit on top of the `your-feature` branch from above.
 $ jj new your-feature
-$ # Address the comments, by updating the code
+$ # Address the comments, by updating the code, then review the changes
```

Comment:
There is an extraneous comma. Also, optionally, I'd split this into two lines.

> $ # Address the comments, by updating the code.
> $ # After that, review the changes

Reply:
I dropped the comma and split it into two sentences. I left it one a single line.